tests/test_tester_utils.py

```diff
@@ -76,3 +81,22 @@ def test_get_last_group(self):
         # Test the case of no match
         last_group = get_last_group(bucket, "TestCam", "20110101")
         self.assertEqual(last_group, int(20110101) * 100_000)
+
+    def test_exposure_id_hsc(self):
+        group = "2023011100026"
```

Comment:
Why not just a number, avoiding the later `int()`?

Reply:
We get a _lot_ of bugs related to integers being assigned as groups and then passed around like that, so I've insisted on other reviews that we always be strict with the types there. I could hardly make an exception for myself, especially since part of the problem is that we tend to _think_ of group IDs as integers when formally they're not.